### PR TITLE
Added troubleshooting page

### PR DESCRIPTION
**Pull request by krishnakatyal**
Signed-off-by: Krishna Katyal krishna.katyal@zilliz.com

### CODE REVIEW

**Comment by fzliu** [comment]
#536

File: docs/get-started/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+
+
+## Troubleshooting
+
+![Example](troubleshooting.jpg)
+### No Module named Towhee
+
+This can be solved by reinstalling the Towhee. To check if Towhee was installed, use the `pip freeze` command and check the installation. Restarting the notebook in case of jupyter notebook,
```
Comment (filip-halt) [inline docs/get-started/troubleshooting.md:14]: comma -> period at the end

Comment (krishnakatyal) [reply docs/get-started/troubleshooting.md]: thank you for pointing out :)

**Comment by fzliu** [comment]
Please rebase (squash) these changes into 1 commit when you are done.

**Pull request closed without merging**